src/odemis/acq/test/fastem_test.py

```diff
@@ -655,6 +655,18 @@ def test_get_abs_stage_movement_overlap(self):
             actual_position = task.get_abs_stage_movement()  # [m]
             numpy.testing.assert_allclose(actual_position, expected_position)
 
+            # Verify that for pre-calibrations compared to the top left corner of the ROA, the stage
```

Comment:
I suggest to make it a separate test case. This test is already extensive and you can just make a simple test for only checking this and only for one overlap.